server/cp/output/formatter/jimi.py

```diff
@@ -116,8 +116,8 @@ class JimiFormatter(Formatter):
 
     ENCODING = "utf-8"
 
-    type = "jimi"
-    name = "Jimi XML"
+    type = "jimi_2"
+    name = "Jimi XML 2"
```

Comment:
seems like you have `jimi_2` defined 2x now

Reply:
resolved in new update